ci/test_java.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+# Copyright (c) 2022-2025, NVIDIA CORPORATION.
+
+set -euo pipefail
+
+. /opt/conda/etc/profile.d/conda.sh
+
+rapids-logger "Downloading artifacts from previous jobs"
+CPP_CHANNEL=$(rapids-download-conda-from-s3 cpp)
+
+rapids-logger "Generate Java testing dependencies"
+
+ENV_YAML_DIR="$(mktemp -d)"
+
+rapids-dependency-file-generator \
+  --output conda \
+  --file-key test_java \
+  --prepend-channel "${CPP_CHANNEL}" \
+  --matrix "cuda=${RAPIDS_CUDA_VERSION%.*};arch=$(arch)" | tee "${ENV_YAML_DIR}/env.yaml"
+
+rapids-mamba-retry env create --yes -f "${ENV_YAML_DIR}/env.yaml" -n test
+
+export CMAKE_GENERATOR=Ninja
```

Comment:
```suggestion
```

Is `CMake` used at all in the Java builds? My understanding was that only `maven` is being invoked and that is never invokes `CMake`.

If I'm right about that, please remove this.

Reply:
Oh good, glad this worked! Thanks for removing it in https://github.com/rapidsai/cuvs/pull/805/commits/88165ad82833f3dabdfb90c0d8ca59196d2f4c7a#diff-76d7916ab418397eef76faf7d4eb5f000cff2b417def935c97f0d2c727afb6af

In the future, it'd be helpful for me as a reviewer if you'd comment on these suggestions saying that you fixed them, and then click "resolve conversation" to collapse the thread.

Reply:
Fixed